src/messages.js

```diff
@@ -0,0 +1,9 @@
+var req = require.context('../lang', true, /\.json.*$/);  
+var exports = {};
+
+req.keys().forEach(function (file) {
```

Comment:
ah, I didn't understand it was a method on webpack. thanks for the clarification.